extension/lib/shopify/collection-repository/command/ListGraphQl.js

```diff
@@ -0,0 +1,89 @@
+const DEFAULT_NUM_PER_PAGE = 250
+
+class ShopifyCollectionRepositoryCommandListGraphQl {
+  /**
+   * @param {ShopifyStorefrontClient} storefrontClient
+   */
+  constructor (storefrontClient) {
+    this._storefrontClient = storefrontClient
+    this._numberPerPage = DEFAULT_NUM_PER_PAGE
```

Comment:
Its better to have this as a separate (optional) parameter with a default value:

constructor (storefrontClient, numberPerPage = DEFAULT_NUM_PER_PAGE) {

@param {number} [numberPerPage=DEFAULT_NUM_PER_PAGE]

[] marks it as optional and assigning DEFAULT_NUM_PER_PAGE declares the default value

Reply:
done